src/lib/components/NotificationModal.svelte

```diff
@@ -69,6 +80,49 @@
         >✕</button
       >
     </form>
+    <!-- Read all notifications (notifications are read on visit otherwise) -->
+    {#if notifications !== undefined && notifications?.filter((n) => n.readAt === null)?.length > 0}
```

Comment:
```suggestion
    {#if notifications && notifications?.filter((n) => n.readAt === null)?.length > 0}
```
This check could be slightly simplified, but I don't think it's too important.